src/com/company/Main.java

```diff
@@ -242,12 +242,12 @@ private static Map<UUID, String> findCustomerIdAndName(UUID id) {
   }
 
   private static Discount findDiscountById(String discountId) throws Exception {
-    for (Discount discount : DISCOUNT_LIST) {
-      if (discount.getId().toString().equals(discountId)) {
-        return discount;
-      }
+    Optional<Discount>discount1=DISCOUNT_LIST.stream().filter(i->i.getId().toString().equals(discountId)).findFirst();
+    if(discount1.isPresent()){
+      return discount1.get();
+    }else{
+      throw new Exception("Discount couldn't applied because couldn't found");
```

Comment:
can you please reformat the code.
discount1 is not a good variable name, can be optDiscount

Reply:
@ihccik  I change it